### ISSUE TITLE
Error installing application: Cannot refresh cache whilst offline None 

### ISSUE THREAD

**Issue by dkuku** [freedombox/Plinth#445]
Hi - trying to install freedombox on my nas 
I first upgarded to debian testing and then installed freedombox-setup, added fbx user with password and at the end i've run /usr/lib/freedombox/setup- everything went smoothly until creating user:
```
×
Creating LDAP user failed.
×
Failed to add new user to admin group.
×
User account created, you are now logged in
Setup Complete!

FreedomBox setup is now complete. To make your FreedomBox functional, you need some applications. They will be installed the first time you access them. 
```
it creates the user and I can login but can't do nothing - when trying to install something i get:
`Creating LDAP user failed.`
and thats the error crom status.log
``` 
[2016-04-24 01:48:51,218] plinth.setup   INFO     Running install for module - shaarli, packages - ['shaarli']
[2016-04-24 01:48:56,478] plinth.setup   ERROR    Error running setup - PackageException(error_string="Cannot refresh cache whilst offline", error_details="None")
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/plinth/package.py", line 89, in install
    self._do_install()
  File "/usr/lib/python3/dist-packages/plinth/package.py", line 103, in _do_install
    False, None, self.progress_callback, self)
GLib.GError: pk-client-error-quark: Cannot refresh cache whilst offline (257)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/plinth/setup.py", line 77, in run
    self.module.setup(self, old_version=current_version)
  File "/usr/lib/python3/dist-packages/plinth/modules/shaarli/__init__.py", line 61, in setup
    helper.install(['shaarli'])
  File "/usr/lib/python3/dist-packages/plinth/setup.py", line 100, in install
    transaction.install()
  File "/usr/lib/python3/dist-packages/plinth/package.py", line 91, in install
    raise PackageException(exception.message) from exception
plinth.package.PackageException: PackageException(error_string="Cannot refresh cache whilst offline", error_details="None")
root@debian:/var/lib/plinth# 
```

**Comment by dkuku** [comment]
I think that for some reason dpkg didn't configure he slapd daemon when installing - I've done this now manually and investigating further

**Comment by dkuku** [comment]
I solved the issue with creating users but still can't install any modules

**Comment by jvalleroy** [comment]
This looks similar to https://github.com/freedombox/Plinth/issues/136. Could you check the network connections on Plinth's Networks page, and see which are active? Do you have network interfaces listed in /etc/network/interfaces or /etc/network/interfaces.d?

**Comment by jvalleroy** [comment]
Fixed in #493.
